bin/aptos

```diff
@@ -1,18 +1,22 @@
 #!/usr/bin/env node
 
-const { spawn } = require("child_process");
+const { execSync, spawn } = require("child_process");
 const fs = require("fs");
 const os = require("os");
 
 const PNAME = "aptos-cli";
 const VERSION = "2.3.1";
 
+const execSyncShell = (command, options) => {
+  return execSync(command, { shell: true, ...options });
+}
+
 // Get all arguments passed to the script
 const getOS = () => {
   const platform = os.platform();
   switch (platform) {
     case "darwin":
-      return "MacOSX";
+      return "MacOS";
```

Comment:
how come?

Reply:
It's not called OS X anymore 😛

Reply:
TIL, it was changed in 2012

Reply:
weird it was working....

Reply:
This is just an internal string.  `darwin` is what it shows up when you ask the CLI what OS it is.